crates/aptos/e2e/local_testnet.py

```diff
@@ -19,6 +19,18 @@ def run_node(network: Network, image_repo_with_project: str):
     container_name = f"aptos-tools-{network}"
     LOG.info(f"Trying to run aptos CLI local testnet from image: {image_name}")
 
+    # Confirm that the Docker daemon is running.
```

Comment:
Down the line we can unify some of this Python, but let's not worry about it for now.